mllib/src/main/scala/org/apache/spark/mllib/rdd/MLPairRDDFunctions.scala

```diff
@@ -49,6 +49,18 @@ class MLPairRDDFunctions[K: ClassTag, V: ClassTag](self: RDD[(K, V)]) extends Se
       }
     ).mapValues(_.toArray.sorted(ord.reverse))  // This is a min-heap, so we reverse the order.
   }
+
+  def topByKeyUsingBuckets(num: Int, bucketsCount: Int = 200)(implicit ord: Ordering[V]): RDD[(K, Array[V])] = {
```

Comment:
does this exist  in scala ? I thought scala does not allow overloading and default argument values are supposed to replace that. let me check ...